### ISSUE TITLE
Support RFC6851 - IMAP MOVE/UID MOVE commands

### ISSUE THREAD

**Issue by hughescr** [lefcha/imapfilter#245]
It'd be nice if imapfilter supported MOVE instead of doing COPY/FLAG/EXPUNGE - for operations where bulk moves are happening, COPY/FLAG/EXPUNGE will result in a ton more I/O (and a lot more disk space required during the copy) on a server that supports MOVE.

https://datatracker.ietf.org/doc/html/rfc6851

**Comment by mh166** [comment]
Added benefit: if you're storage quota is low, it would allow you to still move around a lot of messages at once. Otherwise, you tend to get your wishes denied by the mail server. Ask me how I know... 😉

**Comment by lefcha** [comment]
Yes, that would be nice, I hope I can find some time to look into it...
